### PR TITLE
re-added validator nonce but correctly

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: cli/sign.go [hunk 1]
```diff
@@ -155,26 +155,33 @@ func createSessionID() ([]byte, error) {
...
 	}
 	response, err := client.Sign(data)
 	if err != nil {
 		return api.SignResponse{}, fmt.Errorf("error signing: %w", err)
 	}
 
+	// verify that the signature over the validator nonce verifies to prevent funky replays and such
+	if err = suite.Verify(crypto.ValidatorNonceMessage(validatorNonce), identity.Public, response.DepositValidatorNonceSignature); err != nil {
+		return api.SignResponse{}, fmt.Errorf("signature did not verify for the signed validator nonce for node %s: %w", identity.Address, err)
+	}
```
Comment (AnomalRoil) [inline cli/sign.go]: How is this actually related to the partial signatures verification path? 
I think my mental model around what this `validatorNonce` does is still off.

**Review by AnomalRoil — APPROVED** [review]
Mostly LGTM, just not sure why the verification is in the partial signature verification

File: cli/sign.go (hunk 1 above)
Comment (CluEleSsUK) [reply cli/sign.go]: perhaps `getVerifiedPartial` is a poor name - it really runs a dkg, gets some partial sigs and other related bits

Comment (CluEleSsUK) [reply cli/sign.go]: okay changed the name for clarity
